src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const splited = sourceString.split(';');
```

Comment:
The variable name 'splited' is a misspelling of 'split'. Consider renaming it to 'styleDeclarations' to enhance clarity and correctness.